### ISSUE TITLE
Two trivia directories causes issue

### ISSUE THREAD

**Issue by CinemaVision** [CinemaVision/script.cinemavision#72]
If a **Trivia** and a **Trivia Slides** directory exist, the (usually blank) Trivia directory is used which prevents trivia playback.

**Comment by ruuk** [comment]
Trivia is the correct directory. We have code  that looks for a `Trivia Slides` directory and if found, deletes any `Trivia` directory if one exists then renames `Trivia Slides` to `Trivia`. If it fails to remove the Trivia directory for any reason (a big one being that it is not empty) then there is not much we can do about it. This code is here for migration purposes and probably should be removed now, since it has been so long since the change that required it. At this point it should be a user fix - in this case manually remove the `Trivia` directory and rename `Trivia Slides`.

**Comment by CinemaVision** [comment]
Fair point, well made. Lets go ahead and remove the migration code, but if we can, keep something in the log that alerts us to the presence of the old trivia directory in case we have stragglers.

**Comment by CinemaVision** [comment]
Remove all migration code and logging.
